go.mod

```diff
@@ -30,7 +30,21 @@ require (
 	github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5
 	github.com/aws/aws-lambda-go v1.13.3
 	github.com/aws/aws-sdk-go v1.38.60
-	github.com/aws/aws-sdk-go-v2 v0.24.0
+	github.com/aws/aws-sdk-go-v2 v1.16.2
+	github.com/aws/aws-sdk-go-v2/config v1.15.3
+	github.com/aws/aws-sdk-go-v2/credentials v1.11.2
+	github.com/aws/aws-sdk-go-v2/service/cloudwatch v1.18.1
+	github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs v0.26.0
+	github.com/aws/aws-sdk-go-v2/service/costexplorer v1.18.3
```

Comment:
[service/costexplorer/v1.18.4](https://github.com/aws/aws-sdk-go-v2/releases/tag/service%2Fcostexplorer%2Fv1.18.4) is the latest?